automation/addLabelsOnPullRequest.automation.js

```diff
@@ -65,103 +63,38 @@ exports.addMergedLabel = async context => {
 		const issueLabels = await listIssueLabels(context);
 
 		const foundApproveLabel = issueLabels.data.filter(
-			label => label.name === ':white_check_mark: Approved'
+			label => label.name === ":white_check_mark: Approved"
 		);
 		const foundWIPLabel = issueLabels.data.filter(
-			label => label.name === ':construction: WIP'
+			label => label.name === ":construction: WIP"
 		);
 
-		addLabel([':sparkles: Merged:'], context);
+		addLabel([":sparkles: Merged:"], context);
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹` with `····`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
    addLabel([":sparkles: Merged:"], context);
```